Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.cpp

```diff
@@ -326,8 +326,28 @@ namespace EMotionFX
         {
             return;
         }
+        m_editor->InvalidateAll();
+        if (!m_editor->HasDisplayedNodes())
+        {
+            this->hide();
+            //m_editor->ClearInstances(true);
+        }
+        else
+        {
+            this->show();
+        }
+    }
 
-        m_editor->InvalidateValues();
+    void ColliderWidget::SetFilterString(AZStd::string str)
```

Comment:
nit: Please use more informative variable/parameter names - for example this would be `filterString` not `str`